envio/flow/mappings/common/adjust-stream.ts

```diff
@@ -34,9 +35,15 @@ const handler: Handler = async ({ context, event }) => {
 
   /* --------------------------------- STREAM --------------------------------- */
   const now = BigInt(event.block.timestamp);
-  const elapsedTime = now - stream.lastAdjustmentTimestamp;
-  const streamedAmount = stream.ratePerSecond * elapsedTime;
-  const snapshotAmount = stream.snapshotAmount + streamedAmount;
+  // If the stream has not started yet, the snapshot amount is not updated.
+  let snapshotAmount = stream.snapshotAmount;
+  if (now > stream.startTime) {
+    const streamingStart =
```

Comment:
suggested new name for this variable: `actualStartTime`

Or `resolvedStartTime`.